### PR TITLE
Localize more kubevirt strings

### PR DESCRIPTION
**Pull request by pcbailey**

### CODE REVIEW

**Comment by pcbailey** [comment]
@rawagner Please review

File: frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx [hunk 1]
```diff
@@ -567,18 +590,16 @@ export const UploadPVCPage: React.FC<UploadPVCPageProps> = (props) => {
...
+              <Alert variant="warning" isInline title={t('kubevirt-plugin~File type extension')}>
                 <p>
-                  Based on the file extension it seems like you are trying to upload a file which is
-                  not supported (
-                  {fileNameExtension
-                    ? `detected file extension is '${fileNameExtension}'`
-                    : 'no file extention detected'}
-                  ).
+                  {t(
+                    'Based on the file extension it seems like you are trying to upload a file which is not supported ({{fileNameExtText}}).',
```
Comment (rawagner) [inline frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx:596]: missing kubevirt ns

**Review by rawagner — CHANGES_REQUESTED** [review]

File: frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx (hunk 1 above)
Comment (pcbailey) [reply frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx]: Fixed! =)

**Pull request merged**